FreeRTOS/FreeRTOS
# Add SMP one task enter critical on target test

Add SMP one task enter critical on target test

Description
-----------
Adding the on target test to verify that
> Only one task/ISR shall be able to enter critical section at a time.

Test Steps
-----------
Test result
```
./FreeRTOS/FreeRTOS/Test/Target/tests/smp/only_one_task_enter_critical/only_one_task_enter_critical.c:235:Test_OnlyOneTaskEnterCritical:PASS

-----------------------
1 Tests 0 Failures 0 Ignored
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
